client/gutenberg/editor/calypsoify-iframe.tsx

```diff
@@ -312,10 +316,13 @@ class CalypsoifyIframe extends Component< Props & ConnectedProps & ProtectedForm
 		if ( ! this.iframePort ) {
 			return;
 		}
-
+		if ( this.props.isSiteUsingFSE ) {
+			content = `<!-- wp:a8c/template {"templateId":15,"className":"site-header site-branding"} /-->
+						${ content }
+			            <!-- wp:a8c/template {"templateId":16,"className":"site-footer"} /-->`;
```

Comment:
This line mixes tabs and spaces to indent. Apparently Prettier doesn't work on TS files? 🤔

Reply:
I wonder if it's at all possible to pull this string from the FSE files so it doesn't have to be duplicated 🤔 

https://github.com/Automattic/wp-calypso/blob/6352a65e89f258fc120ce196b31a0658e77751dc/apps/full-site-editing/full-site-editing-plugin/full-site-editing/templates/class-wp-template.php#L151-L153

Though since that's in PHP, it wouldn't be very nice getting it into JS :(

Reply:
I think this is good enough for now.
When/if eventually we start adding more stuff and giving users more freedom, we'll have to change this into a more dynamic thing.
The classes could be fetched from the theme API; the structure could be obtained from a list of page templates, and those could be fetched from another API, etc.

Right now, I guess we _might_ add this to the [`wp_localize_script`](https://github.com/Automattic/wp-calypso/blob/91b2c069f251cbeabef629b3dc192186fd57d31f/apps/full-site-editing/full-site-editing-plugin/full-site-editing/class-full-site-editing.php#L174-L183), but it would sound a bit off to me. 🤔 

For now, though, let's keep things simple, and maybe add a `TODO` comment here and on the [PHP side](https://github.com/Automattic/wp-calypso/blob/6352a65e89f258fc120ce196b31a0658e77751dc/apps/full-site-editing/full-site-editing-plugin/full-site-editing/templates/class-wp-template.php#L151-L153
) to make sure that if we change one we need to also change the other.